packages/core/src/theme.ts

```diff
@@ -573,12 +574,12 @@ export function createButtonsTheme(theme: { ColorPalette: IColorPalette; TextThe
     critical: {
       padding: 16,
       borderRadius: 4,
-      backgroundColor: theme.ColorPalette.brand.primary,
+      backgroundColor: theme.ColorPalette.semantic.error,
     },
     criticalDisabled: {
       padding: 16,
       borderRadius: 4,
-      backgroundColor: theme.ColorPalette.brand.primaryDisabled,
+      backgroundColor: theme.ColorPalette.brand.errorDisabled,
```

Comment:
```suggestion
      backgroundColor: `rgba(216, 41, 47, ${lightOpacity})`,
```

I think it's fine to put it here directly since it would be a net new property that's not expected and isn't used anywhere else